src/main/java/edu/wisc/my/restproxy/dao/GenericRestLookupDao.java

```diff
@@ -4,6 +4,10 @@
 
 import org.springframework.http.HttpMethod;
 
+/**
+ * @deprecated see {@link RestProxyDao} instead
+ */
+@Deprecated
```

Comment:
Since this is a jar component, rather than a service interface, is there value in supporting new and old in the same major version?
I'm wondering if we shouldn't just delete it in this version, since anyone using the deprecated class has no reason to upgrade anyway.